fjord/feedback/tests/test_api.py

```diff
@@ -1,3 +1,5 @@
+# This Python file uses the following encoding: utf-8
+
```

Comment:
What I suggested is the standard way to do it since it's recognized by many editors including but not limited to Emacs.

I'm not sure I've seen the other styles. I didn't even know there were other ways to specify the file encoding.